Software-Engineering.md

```diff
@@ -133,6 +133,24 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I can confidently and charismatically pitch ideas, positively influencing my team to take decisive action based on reaching consensus.
 - I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
 
+## Principal Software Engineer
+
+> I am an influential Engineering [leader](Leadership.md) who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus.
+
+### I have a knack for uncovering important problems and opportunities that are critical to the long-term success of Octopus
+
+- I have a deep understanding of Octopus products and our strategy
```

Comment:
If a principal engineer is working in the Octopus Server core team, how deep does their knowledge need to be in areas covered by other teams?

Reply:
Is there something to be said along the lines of "a principal engineer needs to be across all of the major things happening at Octopus?"

Reply:
```suggestion
- I have a deep understanding of Octopus products and strategy
- I collaborate with other Engineering leaders to ensure I am aware of all major initiatives at Octopus and can account for them in my own
```

Reply:
(Small typo btw? `in my own`)

Reply:
Nope, perhaps it needs the word "initiatives" on the end again...

Reply:
Ah, right 🙈 Yup, I think `initiatives` at the end helps, thanks 🙇